packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts

```diff
@@ -87,6 +88,20 @@ export class LayeredRuntimeResolverCreator {
         }
       }
 
+      // If delta marker is encountered, add resolver for `delta` query
+      if (parseMarker('delta', resolverElement.graphqlType.description)) {
+        const diffQuery = `get${resolverElement.graphqlType.name}Delta`
+
+        resolvers.Query[diffQuery] = (parent: any, args: any, context: any) => {
+          const filter: any = {};
+          if (args?.lastSync) {
+            filter.updatedAt = { gt: args.lastSync};
```

Comment:
This should merge the user filter from args too.

Reply:
Ignore that statement ^ I see there are no filter arguments in the resolver method.

Reply:
Do you think we should have that though? Might be useful

Reply:
@wtrocki would be best to answer this one. I can see it being useful but it may cause some side effects I am unaware of.

Reply:
Yes. Delta should be filtered the same way as the regular find method.